src/write.rs

```diff
@@ -576,6 +753,39 @@ mod tests {
         w.append(CounterAddOp(3));
         assert_eq!(w.oplog.len(), 2);
     }
+    #[test]
+    fn append_test_compress() {
```

Comment:
I'd like to see more read assertions in this test, and not just oplog length assertions. Doesn't matter if the oplog is the right length if the value is wrong ;)

Reply:
Done: f49adcac65e0b22ac2130de393ec1df8f28f0c04